src/encoded/docs/adding_new_assays.md

```diff
@@ -0,0 +1,156 @@
+
+    ```
+    {
+        "_test": "CUT&Tag test",
+        "uuid": "6600481e-b667-46ff-acd8-7de7026c2fdf",
+        "accession": "ENCSR830QHT",
+        "lab": "/labs/john-stamatoyannopoulos/",
+        "award": "U54HG007010",
+        "status": "released",
+        "date_released": "2015-08-31",
+        "biosample_ontology": "tissue_UBERON_0001891",
+        "assay_term_name": "CUT&Tag"
+    }
+    ```
+    * Update existing test features. Now that another experiment is added to our inserts, some tests will break without a couple of corrections. We will need to add our experiment to the number of expected fixtures.
+
+        * Add 1 to the numbered results in [tests/features/views.feature]:
+        ```
+        When I click the link to "/matrix/?type=Experiment"
+        Then I should see "Showing 72 results"
+        ```
+        * Add 1 to the ```len(lines)``` in [tests/test_batch_download.py]:
+        ```
+        def test_batch_download_report_download(testapp, index_workbook):
+        ...
+        assert len(lines) == 72
+        ```
+        * Assess if your new insert will break any other tests- the easiest way to do this is simply to run the tests and interpret any errors that come up. The tests are listed in sets in the [encoded README.md].
+
+    * Add a new audit test. To ensure assays with the new assay term name will be audited correctly, you need to create or edit a fixture (a sample experiment object, similar to your insert) and test that it is audited the same way as experiments of other assay types. For example, if you were to test that your new **CUT&Tag** term was added correctly to ```targetRequiredAssayList```, you can add new lines to the existing test for this audit.
+
+        * In [tests/test_audit_experiment.py], you'll find this test for checking targets are included in experiments that require them:
+
+        ```
+        def test_audit_experiment_target(testapp, base_experiment):
+            testapp.patch_json(base_experiment['@id'], {'assay_term_name': 'ChIP-seq'})
+            res = testapp.get(base_experiment['@id'] + '@@index-data')
+            assert any(error['category'] == 'missing target'
+                for error in collect_audit_errors(res))
+        ```
+
+        * Then, you can patch a fixture (```base_experiment``` is imported in the test above) already used in the audit test to check your new assay term, for example:
+
+        ```
+        testapp.patch_json(base_experiment['@id'], {'assay_term_name': 'CUT&Tag'})
+            res = testapp.get(base_experiment['@id'] + '@@index-data')
+            assert any(error['category'] == 'missing target'
+                for error in collect_audit_errors(res))
+        ```
+        * If you prefer, you can also write a new experiment fixture which you would add to [tests/fixtures/schemas/experiment.py]. For more detailed instructions, please refer to [making_audits.md].
+
+4. Ontology: 
```

Comment:
I actually think probably this section should be added to the beginning, because you really need to make sure you have an ontological term (OBI or NTR) before adding it to the schemas. You also mention in the current first section "Assays are identified by precise ontological terms that we list in a property *assay_term_name*." It's probably important to address how to get the ontological term first.

Reply:
I shifted it to be the first section, clarified name/preferred_name as mentioned in the other comment, and rewrote the section a little to make more sense with the reordering.

Reply:
I reordered the sections to place this first, clarified name/preferred_name as requested in another comment, and did some rewording of the ontology section in particular to make the swap make more sense